src/definitions/behaviors/form.js

```diff
@@ -1823,6 +1824,58 @@ $.fn.form.settings = {
       ;
     },
 
+    // applies rule if another field's value satisfies a condition
+    condition: function(value, ancillary, $module) {
```

Comment:
I have a little difficulty to understand the new syntax...

My understanding is `condition[FIELD_NAME, RULE1, RULE2, ... RULEn]`.
Am I on the right track?
If so, I feel it is a bit strange to treat `FIELD_NAME` and `RULE`s  in the same way.
I suggest a JSON-like syntax like `{ FIELD_NAME: [ RULE ], FIELD_NAME2: [RULE1, RULE2] }` that will cover use case to have conditions on multiple fields.

Reply:
I agree, the current way will be too confusing for users and doesn't allow for expansion of the rule. I think your suggestion of making it JSON like will be our best shot to making it easy to use and easy to improve in the future.

Reply:
@exoego @hammy2899 Hello guys, based on what the author of this issue (#985) writes:
> A built-in rule to validate a form field based on the value from another field of the same form.

I intended to lay out the parameters as follows, basically using the syntax proposed by the issue author:
`condition[FIELD_NAME, VALUE_TO_CHECK_AGAINST, RULE_TO_APPLY]`
which means: "if THAT_FIELD's value is VALUE, then apply RULE to this field". 

Also I intended to set a hard limit on what `RULE_TO_APPLY` can be, namely one single existing rule, on purpose. I was looking to not risk having recursive rules, or excessively complicated config.

With that said I'm absolutely open to improve this. If you think JSON-like syntax is the way to go, I'll change accordingly 👍

Reply:
I think making this JSON like is still the best option we have especially for future support.

Reply:
@hammy2899 please can you set out a small spec of how you envision this being JSON-like, then I'll try to take it from there. Thanks

Reply:
> I have a little difficulty to understand the new syntax...
> 
> My understanding is `condition[FIELD_NAME, RULE1, RULE2, ... RULEn]`.
> Am I on the right track?
> If so, I feel it is a bit strange to treat `FIELD_NAME` and `RULE`s in the flat way.
> 
> I suggest a JSON-like syntax like `{ FIELD_NAME: [RULE], FIELD_NAME2: [RULE1, RULE2] }`.
> This syntax distinguishes `FIELD_NAME` and `RULE`s.
> This will also can cover use-case to have conditions on multiple fields.

Like how @exoego said

Reply:
@vibridi Are you still interested in continuing to adjust the PR ?

Reply:
@lubber-de I'm sorry, I don't think I will be able to work on this at this time. Feel free to close or take over. Thanks!